test_settings.py

```diff
@@ -47,5 +47,7 @@ def root(*args):
 RUNNING_WITH_TEST_SETTINGS = True
 EVENT_TRACKING_BACKENDS = {}
 XAPI_AGENT_IFI_TYPE = 'external_id'
+EVENT_ROUTING_BACKEND_BATCHING_ENABLED = False
+EVENT_ROUTING_BACKEND_BATCH_INTERVAL = 100
```

Comment:
Should we add `EVENT_ROUTING_BACKEND_BATCH_SIZE` here too?

Reply:
On tests, we are overriding it when needed. Don't think we should define it there